sphinx/templates/latex/latex.tex_t

```diff
@@ -30,6 +30,8 @@
 <%= contentsname %>
 <%= numfig_format %>
 <%= pageautorefname %>
+\newcommand{\sphinxVerbatimContinuedText}{\footnotesize(<%= _('continued from previous page') %>)}
+\newcommand{\sphinxVerbatimContinuesText}{\footnotesize(<%= _('continues on next page') %>)}
```

Comment:
should I add string option to ``'sphinxsetup'`` such as ``verbatimcontinued``, ``verbatimcontinues`` ? with current code, user can use ``'preamble'`` key and ``\renewcommand``. With options, I need to then delay the definitions corresponding to the user options to at begin document. The latex file is then less understandable. Or, we move the lines above to before loading sphinx.sty